scripts/common.sh

```diff
@@ -42,6 +42,10 @@ ERR_GOOGLE_APPLICATION_CREDENTIALS_NOT_FOUND=5
 # shellcheck disable=SC2034
 ERR_DIRECTORY_NOT_FOUND=6
 # shellcheck disable=SC2034
+ERR_UNSUPPORTED_OS=7
+# shellcheck disable=SC2034
+ERR_UNSUPPORTED_OS_DESCRIPTION="[Error]Unsupported OS - please run the scripts on Ubuntu or MacOS"
```

Comment:
Remove the ` - please run the scripts on Ubuntu or MacOS` part because this runs on Debian-based systems. We don't want to keep this up to date.

Reply:
removed